### ISSUE TITLE
IPCs can eat food off of forks

### ISSUE THREAD

**Issue by FlattestGuitar** [ParadiseSS13/Paradise#3605]
Just scoop up whatever food you have with a fork and nom on it. Works for IPCs too and I don't think it should...

**Comment by Spacemanspark** [comment]
Ya, this has been a thing for awhile.

**Comment by FlattestGuitar** [comment]
Spark hiding bugs in his basement?!

Execute.

**Comment by Spacemanspark** [comment]
No, I just keep forgetting to report bugs.

**Comment by FalseIncarnate** [comment]
Honestly, utensils probably could be removed without anyone being sad to see them go.

But this is still a bug (even if it's been known for a while unofficially) and deserves to be properly recorded.

**Comment by Gummin** [comment]
Yeah, I've only used a utensils to eat twice.

**Comment by taukausanake** [comment]
I think utensils should see more use. I for one do not condone eating like barbarians but I'd like to see more life simulator, but that's my opinion for suggestions

**Comment by KasparoVy** [comment]
We eat sauced pasta off plates with bare hands

We're RAW space men

**Comment by taukausanake** [comment]
I think more like stuffing your face into the plate as you are holding it with your hand normally

**Comment by NTSAM** [comment]
I thought spessmans just inhale it.

**Comment by VampyrBytes** [comment]
If you're eating off a fork, you pretty much are just inhaling it... forks don't hold the actual food item, just the reagents from it, which are then transferred to you when you use it

**Comment by taukausanake** [comment]
That's reasonable enough as you're doing the same thing when you eat with your hands, correct? So long as it acts the same either with or without a fork

**Comment by VampyrBytes** [comment]
@taukausanake 
Not really. This issue is a symptom of a design flaw (actually 2). Food doesn't do the eating, so it shouldn't be handling the eating. The food item should be passed over to the mob, where it's checked whether that particular mob can eat (can they access their mouth etc), then if possible pass it over to species to handle. That way species can just override `species.eat` rather than having a load of species diet flags which have to be checked in food (snowflake). The fork just makes it worse because by not taking the item and using that it needs all the same checks that food has. And anytime a species with different dietary needs is added more snowflake has to be added to both food and fork.

Another symptom (although probably even less of an issue than this one) is that some food items give you a specific message when you eat them.... you won't get those when you eat off a fork because the message is attached to the food item, not the reagents.
When I get a bit of time I'll probably refactor eating to the method I described above.
tl/dr
If you're having to put in a load of if conditions, it probably means that what you're doing could probably be dealt with cleaner/more efficiently somewhere else
